src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -26,6 +26,8 @@ namespace OpenTelemetry.Metrics
     /// </summary>
     public struct MetricPoint
     {
+        internal readonly AggregatorStore AggregatorStore;
```

Comment:
Maybe would improve readability if we had a separate boolean or something that indicated the MetricPoint was valid? This could be private then. Though, I guess this increases the size of MetricPoint again a bit...

Reply:
What if we did...

```diff
-        internal readonly AggregatorStore AggregatorStore;
+        private readonly AggregatorStore aggregatorStore;
+        public readonly bool IsInitialized => this.aggregatorStore != null;
```

That might achieve the readability without the size hit?

Reply:
@alanwest I just went ahead and made this change. LMK what you think!